salesagility/SuiteCRM
# Remove some incomplete tests and miscellaneous formatting fixes for the unit test suite

## Description
- Remove incomplete tests for SugarBean, incomplete tests cause the test suite to run significantly slower when the state checker runs with `RUN_PER_TEST` for no real reason. They're replaced by a comment listing all the functions that still need tests.
- Formatting fixes that remove excessive whitespace, and a few typo fixes.
- Remove unnecessary StateChecker invocations in a few tests where they aren't used.

## Motivation and Context
Improve unit suite speed when using the state checker, and also formatting and readability of tests.

Honestly the PHP CodeSniffer should just be backported to 7.10 and added to CI, but that's a whole other thing.

## How To Test This
Make sure the tests still pass on Travis and make sure the formatting changes look acceptable.

## Types of changes

### Final checklist
